scripts/install_sc_component_manager_dependencies.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
This script must be part of sc-machine

Reply:
And call this script from `install_sc_machine_dependencies.sh` if you passed in argument `--cm`